src/jquery.quicksearch.js

```diff
@@ -177,7 +184,12 @@
 			}
 			
 			if (noresults) {
-				this.results(false);
+				if(options.onNoResultsFound !== null){
+					options.onNoResultsFound();
+				}else{
+					this.results(false);
+				}
+				
```

Comment:
Why are you not calling `this.results(false);` ? I think it would be better to do it this way:

```js
if ($.isFunction(options.onNoResultsFound)) {
   options.onNoResultsFound();
}
this.results(false);
```

No?